tests/test_benchmark_tf.py

```diff
@@ -0,0 +1,140 @@
+import os
+import tempfile
+import unittest
+from pathlib import Path
+
+from transformers import AutoConfig, is_tf_available
+
+from .utils import require_tf
+
+
+if is_tf_available():
+    import tensorflow as tf
+    from transformers import TensorflowBenchmark, TensorflowBenchmarkArguments
+
+
+@require_tf
```

Comment:
@LysandreJik - these tests are failing in the `run_pytorch` tests even though I added the @require_tf decorator. I'm a bit lost to why this fails in run pytorch